pysb/simulator/__init__.py

```diff
@@ -0,0 +1 @@
+from scipyode import ScipyOdeSimulator
```

Comment:
Should be a relative import -- `from .scipyode ...`. Otherwise you'll get an error on Python 3 due to absolute_import being in effect.